libs/libc/stdio/lib_setvbuf.c

```diff
@@ -133,7 +134,7 @@ int setvbuf(FAR FILE *stream, FAR char *buffer, int mode, size_t size)
 
   /* Return EBADF if the file is not open */
 
-  if (stream->fs_fd < 0)
+  if (fd < 0)
```

Comment:
should we allow for cookie case

Reply:
hmm, what about this? Can we presume that it is not possible to have stream structure without the stream actually being open?

Reply:
It's impossible to handle this type of error reliably. It's caller's responsibility to pass stream which comes from fopen/fdopen/fopencookie.

Reply:
Ok, removed.